client/src/pages/customer-pages/Home/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import SweetAlert from "react-bootstrap-sweetalert";
+
+import {
+  Input,
+  Button,
+  CustomerPageNavBar,
+  Checkbox
+} from "../../../components";
+import "./home.css";
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
Why are u calling `super` and passing `props` to it? You're not using `props` anywhere else, so you don't need to pass it to the `Component` constructor.